### DISCUSSION TITLE
What is .devcontainer/icon.svg used for?

### DISCUSSION THREAD

**Discussion by eighty4** [community/community#45017] [Codespaces] [answered]
### Select Topic Area

Product Feedback

### Body

I see all the official templates have a `.devcontainer/icon.svg` file, but I don't see it used anywhere in the Codespaces interface. Does anyone know where the icon is used?

**Comment by jge162** [comment, marked as answer]
@eighty4 The icon.svg file in the .devcontainer directory is used as the icon for the Codespaces instance in the GitHub Codespaces UI. It is also used in the Remote Development extension for VS Code when you select a codespace to connect to.
